### PR TITLE
Movie sample changes (in progress) -  DO NOT MERGE before Nov 15.

### PR DESCRIPTION
**Pull request by puneith**

### CODE REVIEW

**Comment by jonparrott** [comment]
@puneith can you add "MERGE ON {DATE}" to the title?

**Comment by puneith** [comment]
@jonparrott Good idea :)

**Comment by puneith** [comment]
@jerjou @jonparrott  Please review.

File: language/movie_nl/main.py [hunk 1]
```diff
@@ -318,12 +280,15 @@ def rank_entities(reader, sentiment=None, topn=None, reverse_bool=False):
 
 
 def get_service():
-    """Build a client to the Google Cloud Natural Language API."""
+    """"Build a client to the Google Cloud Natural Language API."""
```
Comment (jonparrott) [inline language/movie_nl/main.py]: extra `"`

File: language/movie_nl/main.py [hunk 2]
```diff
@@ -318,12 +280,15 @@ def rank_entities(reader, sentiment=None, topn=None, reverse_bool=False):
...
 
     credentials = GoogleCredentials.get_application_default()
-
+    scoped_credentials = credentials.create_scoped(
+          ['https://www.googleapis.com/auth/cloud-platform'])
+    http = httplib2.Http()
+    scoped_credentials.authorize(http)
     return discovery.build('language', 'v1',
-                           credentials=credentials,
+                           http=http,
```
Comment (jonparrott) [inline language/movie_nl/main.py]: You should only need to specify `credentials` and not `http`, yeah?

File: language/movie_nl/main_test.py [hunk 3]
```diff
@@ -69,18 +69,18 @@ def test_process_movie_reviews():
     entities = [json.loads(entity) for entity in entities]
 
     # assert sentiments
-    assert sentiments[0].get('sentiment') == 1.0
+    assert sentiments[0].get('sentiment') == 0.9
```
Comment (jonparrott) [inline language/movie_nl/main_test.py]: Can't these values change as the model changes? Should you just assert that sentiments are some non-None value?

Comment (puneith) [inline language/movie_nl/main_test.py]: Yes but they don't happen that frequently. And so above check allows us to see if something really failed.  Non-None will not allow us to detect some issue model or API might have.

File: language/movie_nl/main.py (hunk 1 above)
Comment (puneith) [inline language/movie_nl/main.py]: done

**Comment by puneith** [comment]
@jonparrott discovery public and pending changes made.

**Comment by jonparrott** [comment]
@puneith merging, but please send a follow-up PR to remove httplib2 usage as soon as possible.

**Pull request merged**